README.md

```diff
@@ -1,35 +1,466 @@
+
+  ![Add Prefab to Scene](/docs/images/name13.jpg)
+
+3. Initialize the Swrve Component
+
+  There are two methods to initialize the SDK component. Automatic initialization and manual initialization. In almost all cases, you should you manual initialization and ensure that the "initialise on start" property is not set in the Swrve prefab.
+
+  Set the `app_id` and `api_key` fields of the component to your Swrve App ID and Swrve API Key.
+
+  ```
+  // By default, Swrve stores all customer data and content in our US data center. 
+  //SwrveComponent.Instance.Config.SelectedStack = Swrve.Stack.EU;
+
+  SwrveComponent.Instance.Init(<app_id>, "<api_key>");
+  ```
+
+  You can now compile and run your project in Unity.
+
+
+4. Go to Edit > Project Settings > Script Execution Order and move `SwrveComponent` to a higher priority in the list. This prevents you from getting a `NullReferenceException` that can occur if Swrve is called before the SDK has finished initializing.
+
+### Sending Events ###
+
+```
+SwrveComponent.Instance.SDK.NamedEvent("my_first_event");
+```
+
+When creating custom events, do not use the `swrve.*` or `Swrve.*` namespace for your own events. This is reserved for Swrve use only. Custom event names beginning with `Swrve.` are restricted and cannot be sent.
+
+An event payload can be added and sent with every event. This allows for more detailed reporting around events and funnels. The associated payload should be a dictionary of key/value pairs; it is restricted to string and integer keys and values. There is a maximum cardinality of 500 key-value pairs for this payload per event. This parameter is optional.
+
+```
+Dictionary<string,string> payload = new Dictionary<string,string>() {
+  {"key1", "value1"},
+  {"key2", "value2"}
+};
+SwrveComponent.Instance.SDK.NamedEvent("my_first_event", payload);
+```
+
+### Send User Properties ###
+
+Assign user properties to send the status of the user. For example create a custom user property called `premium`, and then target non-premium users and premium users in the dashboard..
+
+```
+Dictionary<string, string> attributes = new Dictionary<string, string>();
+attributes.Add("premium", "true");
+attributes.Add("level", "12");
+attributes.Add("balance", "999");
+SwrveComponent.Instance.SDK.UserUpdate(attributes);
+```
+
+### Sending Virtual Economy Events ###
+
+If your app has a virtual economy, send the purchase event when users purchase in-app items with virtual currency.
+
+```
+string item = "some.item";
+string currency = "gold";
+int cost = 99;
+int quantity = 1;
+SwrveComponent.Instance.SDK.Purchase(item, currency, cost, quantity);
+```
+
+Send the currency given event when you give users virtual currency. Examples include initial currency balances, retention bonuses and level-complete rewards.
+
+```
+string givenCurrency = "gold";
+double givenAmount = 99;
+SwrveComponent.Instance.SDK.CurrencyGiven(givenCurrency, givenAmount);
+```
+
```

Comment:
Edit.

Before you can send a virtual economy event you need to add the virtual currency name to the Swrve Dashboard.  For example, add "gold" to the dashboard and then use that same string in your game.  If the strings differ, including case, virtual economy events will be rejected.